gtsfm/frontend/verifier/gric_verifier.py

```diff
@@ -0,0 +1,191 @@
+"""
+Wrapper about COLMAP's GRIC estimators, using pycolmap's pybind API.
+Uses LORANSAC for Essential, Fundamental, and Homography matrix estimation.
+
+LORANSAC paper:
+ftp://cmp.felk.cvut.cz/pub/cmp/articles/matas/chum-dagm03.pdf
+
+On Linux and Mac, a python wheel is available:
+https://pypi.org/project/pycolmap/#files
```

Comment:
Can you explicitly mention this verifier is not available on windows?

Reply:
Sure, just added an inline comment with that info.